spec/api-browser-window-spec.js

```diff
@@ -1145,8 +1145,11 @@ describe('BrowserWindow module', function () {
         const initialWebContents = webContents.getAllWebContents()
         ipcRenderer.send('prevent-next-new-window', w.webContents.id)
         w.webContents.once('new-window', () => {
-          assert.deepEqual(webContents.getAllWebContents(), initialWebContents)
-          done()
+          // We need to give it some time so the windows get properly disposed (at least on OSX).
+          setTimeout(() => {
+            assert.deepEqual(webContents.getAllWebContents(), initialWebContents)
+            done()
+          }, 100)
```

Comment:
@kevinsawicki apparently the very same test also fails for the pull request https://github.com/electron/electron/pull/9610 , so maybe the test did not "fail" before just because there were no changes that made the test retrigger?